### DISCUSSION TITLE
Trying to run a reoccurring job WithChronSchedule

### DISCUSSION THREAD

**Discussion by s3YwCf2ZbfJG4SHAfjQMAjtsf** [quartznet/quartznet#1825] [Q&A]
chron = 0 0/1 0 ? * * *

I want to run the job once a minute.   The chron will be dynamic from an Outlook meeting interface.   I used this site to generate my chron string:
https://www.freeformatter.com/cron-expression-generator-quartz.html#cronconverttotext

I was trying to test multiple job runs with different X minutes intervals..   Not sure what I am missing?   

```
                IJobDetail job = JobBuilder.Create<HsMergeJob>()
                    .WithIdentity(jobName, JobGroup)
                    .WithDescription(jobName)
                    .DisallowConcurrentExecution(true)
                    .Build();

                ITrigger trigger = TriggerBuilder.Create()
                    .WithIdentity(jobName, JobGroup)
                    .StartNow()
                    .WithDescription($"Executing {jobName} Notification")
                    .WithCronSchedule(chron)
                    .Build();

                await _scheduler.ScheduleJob(job, trigger);
```

Thanks

**Comment by s3YwCf2ZbfJG4SHAfjQMAjtsf** [comment]
Apparently, the syntax from that web site is not correct.   I think the ? is ending up in the wrong place.

The Quartz.Net site has the following example:  0 0/5 * * * ? for every 5 minutes.

**Reply by lahma** [reply]
Try http://www.cronmaker.com/ , Quartz expects first one to be seconds.
